app/components/table.gts

```diff
@@ -151,6 +74,23 @@ export default class Table extends Component<{}> {
   agGridElement?: HTMLElement;
   agGridInstance?: Grid;
 
+  // Instances of "ember cell" to render (visible in viewport or in buffer)
+  @tracked emberCellRenderers: EmberCellRenderer[] = [];
+
+  // There's no easy way to get TooltipRenderer instance when a custom tooltip is shown (like we do through `getCellRendererInstances` for custom cells)
+  // This is a workarround allowing a custom tooltip to "register itself" before there are actually rendered.
+  // It allows table component to properly render ember component as tooltip
+  //
+  // This workarround (+ usage of "data-ember-tooltip" attribute) could get away if :
+  // - `onTooltipShow` event could include the TooltipComponent instance
+  // or
+  // - a kind of "getTooltipRendererInstance" method is added to Grid API
+  readonly context = {
+    activeTooltipRender: undefined,
```

Comment:
Definitely not ideal way of dealing with the `TooltipRenderer` but, adding TooltipComponent instance on `onTooltipShow` event or add `getTooltipRendererInstance` to grid API looks like something doable via a contribution to ag-grid 🤔